components/scroller/scroller_default.story.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div>
+    <h3>Scroller</h3>
```

Comment:
We should only put one scroller on the default page that can be changed via the props that are set in the controls. If we want to show more than one example with static props like you are doing here, make a `scroll_variants.story.vue` page.

Reply:
ye it was in TO DO, but now let me build it, I will build also the variant (dynamic-size)